cmd/statusd/main.go

```diff
@@ -67,6 +87,30 @@ func main() {
 	node.Wait()
 }
 
+// startDebug starts the debugging API server.
+func startDebug(backend *api.StatusBackend) error {
+	statusAPI := api.NewStatusAPIWithBackend(backend)
+	_, err := debug.New(statusAPI, *cliPort)
+	return err
+}
+
+// startPprof starts the PPROF endpoints on the defined port.
+// Manual registration is needed to only start on demand and
+// on the configured port.
+func startPprof() {
+	mux := http.NewServeMux()
+	mux.HandleFunc("/debug/pprof/", pprof.Index)
+	mux.HandleFunc("/debug/pprof/cmdline", pprof.Cmdline)
+	mux.HandleFunc("/debug/pprof/profile", pprof.Profile)
+	mux.HandleFunc("/debug/pprof/symbol", pprof.Symbol)
+	mux.HandleFunc("/debug/pprof/trace", pprof.Trace)
+	s := http.Server{
+		Addr:    ":" + (*pprofPort),
+		Handler: mux,
+	}
+	go s.ListenAndServe() // nolint: errcheck
+}
```

Comment:
If this PR introduces also profiling related code, it should probably remove `helpers/profiling` as it redundant. Or, do it in separate PR, whichever makes more sense.

Reply:
Will remove it to create extra PR for it.